python/aswfdocker/tests/test_migrater.py

```diff
@@ -0,0 +1,83 @@
+import unittest
+import logging
+
+from click.testing import CliRunner
+
+from aswfdocker import migrater, constants
+from aswfdocker.cli import aswfdocker
+
+
+class TestMigrater(unittest.TestCase):
+    def setUp(self):
+        self.maxDiff = None
+
+    def test_migrate_nofilter(self):
+        m = migrater.Migrater("src", "dst")
+        m.gather("", "")
+        self.assertGreater(len(m.migration_list), 10)
+
+    def test_migrate_pkgfilter(self):
+        m = migrater.Migrater("src", "dst")
+        m.gather("openexr", "")
+        self.assertEqual(len(m.migration_list), 3)
+
+    def test_migrate_versionfilter(self):
+        m = migrater.Migrater("src", "dst")
+        m.gather("openexr", "2019")
+        current_version = constants.VERSIONS[constants.ImageType.PACKAGE]["openexr"][1]
+        self.assertEqual(len(m.migration_list), 1)
+        minfo = m.migration_list[0]
+        self.assertEqual(minfo.image, "ci-package-openexr")
+        self.assertEqual(minfo.version, "2019.1")
+        self.assertEqual(
+            minfo.source, f"docker.io/src/ci-package-openexr:{current_version}"
+        )
+        self.assertEqual(
+            minfo.destination, f"docker.io/dst/ci-package-openexr:{current_version}"
+        )
+
+        m.migrate(dry_run=True)
+        self.assertEqual(
+            m.cmds,
+            [
+                f"docker pull docker.io/src/ci-package-openexr:{current_version}",
```

Comment:
As with earlier comment, hostname of Docker repository should come from config file if we want to move to potential other Docker repository.